.cargo/config.toml

```diff
@@ -1,3 +1,7 @@
 [build]
 # Share one `target` directory at the project root for all Cargo projects and workspaces in smithy-rs
 target-dir = "target"
+
+# TODO(Rust 1.70): The sparse registry config can be removed when upgrading to Rust 1.70
```

Comment:
We leave links in the TODO description usually and it's probably a better way to remember them. I cut it for you: https://github.com/awslabs/smithy-rs/issues/2766